app/uk/gov/hmrc/thirdpartyapplication/connector/AwsApiGatewayConnector.scala

```diff
@@ -52,8 +52,6 @@ class AwsApiGatewayConnector @Inject()(http: HttpClient, config: AwsApiGatewayCo
       val requestId = (result.json \ "RequestId").as[String]
       Logger.info(s"Successfully created or updated application '$applicationName' in AWS API Gateway with request ID $requestId")
       HasSucceeded
-    } recover {
-      awsRecovery(s"Failed to create or update application '$applicationName' in AWS API Gateway")
```

Comment:
Was awsRecovery used elsewhere and if not was it removed altogether ?

Reply:
Yeah it was a private method in this class, used in 2 places, now gone completely.

Reply:
The idea was to have it until we migrated from WSO2 as it was dual running for a while and we didn't want failures in AWS to cause any issues while still on WSO2.